<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filters_Outliers</class>
 <widget class="QWidget" name="Filters_Outliers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="Radius_Remove_Outliers">
     <property name="title">
      <string>Radius Outliers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QSpinBox" name="ROR_MinNeighborsInRadius">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>MinNeighborsInRadius</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>RadisuSearch</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="ROR_Inverse">
        <property name="text">
         <string>Inverse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="Remove_Radius_Outliers">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="ROR_RadiusSearch">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="Remove_Radius_Outliers_ALL">
        <property name="text">
         <string>RemoveALL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="Statistical_Remove_Outliers">
     <property name="title">
      <string>Statistical Outliers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QCheckBox" name="SOR_Inverse">
        <property name="text">
         <string>Inverse</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="Remove_Statistical_Outliers">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MeanK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Std Min Threshold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SOR_MeanK">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="SOR_StddevMulThresh">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="Remove_Statistical_Outliers_ALL">
        <property name="text">
         <string>RemoveALL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ROR_MinNeighborsInRadius</tabstop>
  <tabstop>ROR_RadiusSearch</tabstop>
  <tabstop>ROR_Inverse</tabstop>
  <tabstop>Remove_Radius_Outliers</tabstop>
  <tabstop>SOR_MeanK</tabstop>
  <tabstop>SOR_StddevMulThresh</tabstop>
  <tabstop>SOR_Inverse</tabstop>
  <tabstop>Remove_Statistical_Outliers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>Start_Elab_ROI()</slot>
 </slots>
</ui>
